ERCS/erc-7734.md

```diff
@@ -0,0 +1,153 @@
+---
+eip: 7734
+title: Decentralized Identity Verification (DID)
+description: A standard for decentralized identity verification on the Ethereum blockchain.
```

Comment:
You don't need to mention Ethereum in the title or description. We assume all ERCs are Ethereum related.